lib/codegen/meta-python/isa/x86/recipes.py

```diff
@@ -1473,6 +1486,36 @@ def floatccs(iform):
         disp4(destination, func, sink);
         ''')
 
+indirect_jmp = TailRecipe(
+        'indirect_jmp', IndirectJump, size=1, ins=GPR, outs=(),
+        clobbers_flags=False,
+        emit='''
+        PUT_OP(bits, rex1(in_reg0), sink);
+        modrm_r_bits(in_reg0, bits, sink);
+        ''')
+
+jt_entry = TailRecipe(
+        'jt_entry', BranchTableEntry, size=2,
+        ins=(GPR_DEREF_SAFE, GPR_ZERO_DEREF_SAFE),
+        outs=(GPR),
+        clobbers_flags=False,
+        instp=valid_scale(BranchTableEntry),
+        emit='''
+        PUT_OP(bits, rex3(in_reg1, out_reg0, in_reg0), sink);
+        modrm_sib(out_reg0, sink);
+        sib(imm.trailing_zeros() as u8, in_reg0, in_reg1, sink);
+        ''')
+
+jt_base = TailRecipe(
+        'jt_base', BranchTableBase, size=5, ins=(), outs=(GPR),
+        clobbers_flags=False,
+        emit='''
+        PUT_OP(bits, rex2(0, out_reg0), sink);
+        modrm_riprel(out_reg0, sink);
```

Comment:
Can you add a comment here mentioning that we don't need a reloc because we put the jump table immediately after the function body so we can compute its offset ourselves?